webpack/ForemanTasks/Components/TasksDashboard/Components/TasksCardsGrid/Components/StoppedTasksCard/StoppedTasksCard.js

```diff
@@ -46,45 +49,54 @@ const StoppedTasksCard = ({
-              <tr className={`${result}-row`} key={result}>
-                <td>{capitalize(result)}</td>
-                <td
-                  className={classNames('total-col', {
-                    active:
-                      query.state === STOPPED &&
-                      query.result === result &&
-                      query.mode !== LAST,
-                  })}
-                  onClick={() => updateQuery({ state: STOPPED, result })}
-                >
-                  {total}
-                </td>
-                <td
-                  className={classNames('last-col', {
-                    active:
-                      query.state === STOPPED &&
-                      query.result === result &&
-                      query.mode === LAST &&
-                      query.time === time,
-                  })}
-                  onClick={() =>
-                    updateQuery({
-                      state: STOPPED,
-                      result,
-                      mode: LAST,
-                      time,
-                    })
-                  }
-                >
-                  {last}
-                </td>
-              </tr>
-            ))}
+            {Object.entries(data).map(([result, { total, last }]) => {
+              const active = query.state === STOPPED && query.result === result;
+              const activeTotal = active && query.mode !== LAST;
+              const activeLast =
+                active && query.mode === LAST && query.time === time;
+              const notFocusedTotal =
+                query.state &&
+                !(query.state === STOPPED && !query.result) &&
+                !activeTotal;
+              const notFocusedLast =
+                query.state &&
+                !(query.state === STOPPED && !query.result) &&
+                !activeLast;
+              return (
+                <tr className={`${result}-row`} key={result}>
+                  <td>{capitalize(result)}</td>
+                  <td
+                    className={classNames('total-col', {
+                      active: activeTotal,
+                      'not-focused': notFocusedTotal,
+                    })}
+                    onClick={() => updateQuery({ state: STOPPED, result })}
+                  >
+                    {total}
+                  </td>
+                  <td
+                    className={classNames('last-col', {
+                      active: activeLast,
+                      'not-focused': notFocusedLast,
+                    })}
+                    onClick={() =>
+                      updateQuery({
+                        state: STOPPED,
+                        result,
+                        mode: LAST,
+                        time,
+                      })
+                    }
+                  >
+                    {last}
+                  </td>
+                </tr>
+              );
+            })}
```

Comment:
could the data already load when the page refreshes, instead of starting with 0 and than update?
if it is a problem for now, I am fine with adding it in a follow up PR :)

![Apr 8, 2019 2_59 PM](https://user-images.githubusercontent.com/26363699/55722487-25ff2c00-5a0f-11e9-85b4-be56b510f70b.gif)

Reply:
Yes it makes sense but I would prefer to implement SPA and get this feature automatically with a future PR instead implementing it for a none SPA and then remove it when we introduce SPA.